proactive-jupyter-kernel/kernel.py

```diff
@@ -133,6 +135,11 @@ def __start_proactive__(self):
 
         proactive_host = 'try.activeeon.com'
         proactive_port = '8080'
+
+        self.proactive_config['proactive_server'] = {}
```

Comment:
Can we have a constant somewhere for 'proactive_server'? I dont know it it is possible in Python tho

Reply:
Well, before your question I thought that the answer was no, but I checked to be sure and I found a way to do it. An "ugly" way, but a way lol. I will see if it is a good idea to adapt it.